src/main/java/racing/RacingCars.java

```diff
@@ -24,16 +28,22 @@ public List<Car> getCarList() {
 
     public void startRace() throws CarLocationException {
         for (int i = 0; i < this.carList.size(); i++) {
-            oneCarRace(i);
+            oneCarRace(i, carList.get(i));
         }
     }
 
-    private void oneCarRace(int idx) throws CarLocationException {
+    private void oneCarRace(int idx, Car car) throws CarLocationException {
         Integer randomNumber = RandomNumberGenerator.getRandomNumber();
         if (RacingValidator.isMovable(randomNumber)) {
```

Comment:
```suggestion
        if (randomNumber >= MOVE_FORWARD_CONDITION_NUMBER) {
```
현재 단계에서는 검증 코드가 객체와 멀리 떨어트려놓을 필요가 있을까요?